dir.props

```diff
@@ -272,6 +272,8 @@
 
     <!-- Constructed shared fx path for testing -->
     <TestSharedFxDir Condition="'$(BinPlaceTestSharedFramework)' == 'true'">$(NETCoreAppTestRootPath)</TestSharedFxDir>
+    <TestSharedFxDir Condition="'$(BuildingUAPAOTVertical)' == 'true'">$(ILCFXInputFolder)</TestSharedFxDir>
+    <UseDotNetNativeToolchain Condition="'$(BuildingUAPAOTVertical)' == 'true'">true</UseDotNetNativeToolchain>
```

Comment:
Nevermind I found it :)

We should probably also gate this on the TestILCFolder.